src/core/project/qgsprojectfiletransform.cpp

```diff
@@ -304,7 +305,7 @@ void transform0100to0110( QgsProjectFileTransform *pft )
 {
   if ( ! pft->dom().isNull() )
   {
-#ifndef QT_NO_PRINTER
+#if defined( HAVE_QTPRINTER )
```

Comment:
Is this code REALLY for upgrading 1.0 projects to 1.1? We should kill it entirely if so... !

Reply:
@nyalldawson , removing the function entirely or just the #if block ?

Reply:
I'd just remove it all. We've already required users update 1.0 projects to 2.x before allowing them to be imported into QGIS 3, so it's redundant.

Reply:
@nyalldawson , ok, I'm happy to execute that in a separate PR then.